specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Remove_None_when_redundant.cs

```diff
@@ -0,0 +1,31 @@
+namespace Rules.MS_Build.Remove_None_when_redundant;
+
+public class Reports
+{
+    [Test]
+    public void on_not_alphabetical_order() => new RemoveNoneWhenRedundant()
+       .ForInlineCsproj(@"
+<Project Sdk=""Microsoft.NET.Sdk"">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include=""*"" />
+    <None Remove=""License.md"" />
```

Comment:
Is this truely redundant? doesn't this include everything _except_ `License.md` or does it not work like that?

Reply:
It will have no influence on the output, as `<None>` has no influence. If hiding was intended, that can be solved otherwise, so yes, this should be redundant.